### PR TITLE
updated PWA glossary definition

### PR DESCRIPTION
**Pull request by estelle**
Went in to improve the see also links
Removed marketing creepiness from the definition

### CODE REVIEW

File: files/en-us/glossary/progressive_web_apps/index.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ page-type: glossary-definition
...
+Progressive web apps is a term used to describe the modern state of web app development. This involves taking standard websites/apps that enjoy all the best parts of the Web — such as discoverability via search engines, being linkable via {{Glossary("URL")}}s, and working across multiple form factors — and progressively enhancing them with modern APIs (such as [Service Workers](/en-US/docs/Web/API/Service_Worker_API) and [Push](/en-US/docs/Web/API/Push_API)), enabling features that confer benefits more commonly attributed to native apps.
 
-These features include being installable, working offline, and being easy to sync with and re-engage the user from the server.
+These features include being installable, working offline, and being easy to sync with and re-engaging the user from the server.
```
Comment (hamishwillee) [inline files/en-us/glossary/progressive_web_apps/index.md:11]: Revert?

File: files/en-us/glossary/progressive_web_apps/index.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ page-type: glossary-definition
 
 {{GlossarySidebar}}
 
-Progressive web apps is a term used to describe the modern state of web app development. This involves taking standard websites/apps that enjoy all the best parts of the Web — such as discoverability via search engines, being linkable via {{Glossary("URL")}}s, and working across multiple form factors — and supercharging them with modern APIs (such as [Service Workers](/en-US/docs/Web/API/Service_Worker_API) and [Push](/en-US/docs/Web/API/Push_API)) and features that confer other benefits more commonly attributed to native apps.
+Progressive web apps is a term used to describe the modern state of web app development. This involves taking standard websites/apps that enjoy all the best parts of the Web — such as discoverability via search engines, being linkable via {{Glossary("URL")}}s, and working across multiple form factors — and progressively enhancing them with modern APIs (such as [Service Workers](/en-US/docs/Web/API/Service_Worker_API) and [Push](/en-US/docs/Web/API/Push_API)), enabling features that confer benefits more commonly attributed to native apps.
```
Comment (hamishwillee) [inline files/en-us/glossary/progressive_web_apps/index.md:9]: This is obviously better. My only concern is the pre-existing first sentence:

>Progressive web apps is a term used to describe the modern state of web app development. 

I hate the use of modern, as it ages badly :-). Also it is just one possible aspect of web app development. I think we should steal the first sentence from the new PWA docs.

I'm not attached to the words, but you get the point:

```suggestion
Progressive web applications (PWA) are applications that are built using web platform technologies, but that provide a user experience like that of a platform-specific app.

These kinds of apps enjoy enjoy all the best parts of the Web — such as discoverability via search engines, being linkable via {{Glossary("URL")}}s, and working across multiple form factors, but are progressively enhanced with modern APIs (such as [Service Workers](/en-US/docs/Web/API/Service_Worker_API) and [Push](/en-US/docs/Web/API/Push_API)).
```

File: files/en-us/glossary/progressive_web_apps/index.md [hunk 3]
```diff
@@ -6,11 +6,13 @@ page-type: glossary-definition
...
+Progressive web applications (PWA) are applications that are built using web platform technologies, but that provide a user experience like that of a platform-specific app.
 
-These features include being installable, working offline, and being easy to sync with and re-engage the user from the server.
+These kinds of apps enjoy enjoy all the best parts of the Web — such as discoverability via search engines, being linkable via {{Glossary("URL")}}s, and working across multiple form factors, but are progressively enhanced with modern APIs (such as [Service Workers](/en-US/docs/Web/API/Service_Worker_API) and [Push](/en-US/docs/Web/API/Push_API)).
+
+These features include being installable, working offline, and being easy to sync with and re-engaging the user from the server.
```
Comment (estelle) [inline files/en-us/glossary/progressive_web_apps/index.md:13]:
```suggestion
These features include being installable, working offline, and being easy to sync with and re-engage the user from the server.
```

**Review by hamishwillee — APPROVED** [review]
Thanks!

**Pull request merged**